docs/store/stream-metrics.md

```diff
@@ -0,0 +1,116 @@
+<!--
+title: "Stream metrics between nodes"
+description: ""
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/store/stream-metrics.md
+-->
+
+# Stream metrics between nodes
+
+Each node running the Netdata Agent can stream the metrics it collects, in real time, to another node. Streaming allows
+you to replicate metrics data across multiple nodes, or centralize all your metrics data into a single time-series
+database (TSDB).
+
+The node that receives metrics (a **parent**) can visualize metrics from the sending nodes (a **child**), run health
+checks to [trigger alarms](/docs/monitor/view-active-alarms.md) and [send
+notifications](/docs/monitor/enable-notifications), and [export](/docs/export/external-databases.md) all metrics to an
+external TSDB.
+
+Streaming lets you decide exactly how you want to store and maintain metrics data. While we believe Netdata's
+[distributed architecture](/docs/store/distributed-data-architecture.md) is ideal for speed and scale, streaming
+provides centralization options for those who want to maintain only a single TSDB instance.
+
+## Streaming quickstart (`parent-child`)
+
+The simplest streaming configuration is **replication**, in which a child node streams its metrics in real time to a
+parent node, and both nodes retain metrics in their own databases.
+
+See the [streaming reference doc](/streaming/README.md#supported-streaming-configurations) for details about other
+possible configurations.
+
+### Enable streaming on the parent node
+
+First, log onto the node that will act as the parent.
+
+Run `uuidgen` to create a new API key, which is a randomly-generated machine GUID the Netdata Agent uses to identify
+itself while initiating a streaming connection. Copy that into a separate text file for later use.
```

Comment:
I remember that the program does not exist by default on machines. Perhaps we can add something:
> In case you don't have it installed, you will need to install it. For example, on a Linux system that uses the `apt` package manager, you would run `apt-get install uuid-runtime`